bin/cfg/Tomb1Main.json5

```diff
@@ -24,6 +24,9 @@
     // Draws flame when firing a shotgun, like for other guns.
     "enable_shotgun_flash": true,
 
+    // Shotgun no longer fires when a locked target moves out of Lara's sight, like for other guns.
+    "enable_shotgun_target_fix": true,
```

Comment:
I think we should call this `fix_shortgun_targeting` to be consistent with the other OG fix configuration switches.